pkg/controller/admissionchecks/provisioning/controller.go

```diff
@@ -302,6 +303,10 @@ func (c *Controller) syncOwnedProvisionRequest(ctx context.Context, wl *kueue.Wo
 			}
 
 			if err := c.client.Create(ctx, req); err != nil {
+				ac.Message = fmt.Sprintf("Error creating ProvisioningRequest %q: %v", requestName, err)
+				workload.SetAdmissionCheckState(&wl.Status.AdmissionChecks, *ac)
+
+				c.record.Eventf(wl, corev1.EventTypeWarning, "FailedCreate", "Error creating ProvisioningRequest %q: %v", req.Name, err)
```

Comment:
```suggestion
				c.record.Eventf(wl, corev1.EventTypeWarning, "FailedCreate", ac.Message)
```
nit: to avoid code duplication to compute the message.

Reply:
Done.